DNN Platform/Library/UI/Skins/Skin.cs

```diff
@@ -425,7 +444,7 @@ private bool ProcessMasterModules()
                 // Versioning checks.
                 if (!TabController.CurrentPage.HasAVisibleVersion)
                 {
-                    Response.Redirect(Globals.NavigateURL(PortalSettings.ErrorPage404, string.Empty, "status=404"));
+                    HandleAccesDenied(true);
```

Comment:
Hm, you created this method with a parameter but this is the only usage?

Reply:
I propose to replace direct call to Response.Redirect with that method, if you agree

Reply:
I don't mind a method. What I mean is that you added the `bool redirect = false` parameter but this is the only usage, right? It will never be called otherwise. So maybe it should be paremeterless instead if we don't need it now?

Reply:
On lines 455 and 478, error message is displayed without redirect.

Reply:
Oh, ok. Then we should replace them with a call to `HandleAccesDenied`, right?

Reply:
Just pushed ;)